GREYErrorFormatter.m

```diff
@@ -0,0 +1,136 @@
+//
+// Copyright 2017 Google Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+#import "GREYErrorFormatter.h"
+
+#import "GREYObjectFormatter.h"
+#import "GREYInteraction.h"
+#import "GREYErrorConstants.h"
+#import "NSError+GREYCommon.h"
+#import "GREYError+Private.h"
+
+@interface GREYErrorFormatter ()
+
```

Comment:
rm empty lines. Comment the @interface.

Reply:
I'm not sure what comment to leave. Shouldn't the comment be in the .h interface instead?